src/transformers/modeling_utils.py

```diff
@@ -1204,6 +1215,74 @@ def _from_config(cls, config, **kwargs):
 
         return model
 
+    def use_attn_implementation(self, attn_implementation: str):
```

Comment:
```suggestion
    def set_attn_implementation(self, attn_implementation: str):
```
set or switch sound good to me. Also if not implemented we can just remove it?

Reply:
Removed, was just a gist